<?xml version="1.0"?>
<launch>

	<!--<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true" launch-prefix="xterm -e gdb args"> -->
        <node pkg="move_object_actions" type="push_action_node" respawn="false" name="push_action" output="screen" clear_params="true" launch-prefix="xterm -e gdb --args">
        <!--<node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true" > -->
       
    		<rosparam file="$(find semantic_planner_global)/cfg/costmap_common_params.yaml" command="load" ns="global_costmap" />
		<rosparam file="$(find semantic_planner_global)/cfg/global_costmap_params.yaml" command="load" />

		<rosparam file="$(find semantic_planner_local)/cfg/costmap_common_params.yaml" command="load" ns="local_costmap" />
		<rosparam file="$(find semantic_planner_local)/cfg/local_costmap_params.yaml" command="load" />

		<rosparam file="$(find move_object_actions)/config/base_local_planner_params.yaml" command="load" /> 

		
		<remap from="/push_action/global_costmap/costmap" to="/global_costmap/costmap" />
                <remap from="/push_action/global_costmap/costmap_updates" to="/global_costmap/costmap_updates" />
               
  	</node>

       


</launch>

